components/src/dependencies.js

```diff
@@ -58,7 +58,7 @@ export const dependencies = {
 export const paths = {
   tex: '[mathjax]/input/tex/extensions',
   mml: '[mathjax]/input/mml/extensions',
-  sre: '[mathjax]/sre/' + (typeof window === 'undefined' ? 'sre-node' : 'sre_browser')
+  sre: '[mathjax]/sre'
```

Comment:
Since this is only used to get the math maps, I'm wondering if `sre` is the best name to use.  Maybe `mathmaps` or `sre-maps` or something might be better, and it could be set to `[mathjax]/sre/mathmaps`.  This would require corresponding changes to `sre_config.js` above (lines 8 and 9).

Reply:
This is what I effectively have been talking about yesterday. Had not seen your comment then. But would renaming `sre` not be considered a breaking change?

Reply:
The `sre` path now maps to `mathmaps` directory. `sre_config.js` is adapted accordingly.

Reply:
> But would renaming sre not be considered a breaking change?

Technically, so is removing the `sre_browser.js` and `sre-node.js` files that this was used to access.  This value wasn't really used publicly, so it should be OK.  The `[sre]` path was used to obtain the proper version of SRE for browser or node (not the directory itself, but the actual file), and that is no longer necessary, so the change should be a non-issue for users.